### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by taron19**
added method and msg

### CODE REVIEW

File: src/main/java/Test.java [hunk 1]
```diff
@@ -17,6 +17,6 @@ void third(){
     }
 
     void fourth(){
-
+       System.out.println("delete");
```
Comment (taron1991) [inline src/main/java/Test.java]: delete this msg and write hello instead

File: src/main/java/Test.java [hunk 2]
```diff
@@ -17,6 +17,6 @@ void third(){
     }
```
Comment (taron1991) [inline src/main/java/Test.java:18]: got?

**Review by taron1991 — COMMENTED** [review]
change

File: src/main/java/Test.java (hunk 1 above)
Comment (taron19) [reply src/main/java/Test.java]: ok sir

**Comment by taron19** [comment]
added hello

**Review by taron1991 — APPROVED** [review]

**Pull request merged**